antenna/throttler.py

```diff
@@ -245,7 +245,7 @@ def always_match(crash):
     Rule(
         rule_name='is_thunderbird_seamonkey',
         key='ProductName',
-        condition=lambda x: x[0] in 'TSC',
+        condition=lambda x: x and x[0] in 'TSC',
```

Comment:
This enforces that x is a truthy value. So None (it shouldn't ever be None at this point anyhow because of the way the throttler works) and empty string are falsey, so this will return False in those situations rather than kicking up an IndexError because there is no first character.